### ISSUE TITLE
Add command line executable for *nix system and scripting support [feedback requested]

### ISSUE THREAD

**Issue by chrissimpkins** [chrissimpkins/Crunch#4]

**Comment by chrissimpkins** [comment]
Considering the following and could use feedback from those who might desire a command line approach that works on *nix platforms (including Linux, macOS, and Windows nix-like flavors such as MinGW):

- make based install of a shell script executable (`crunch`) on the system PATH
- local (off of system PATH) builds of pngquant and zopflipng from source, supported through make targets so that we can pin the versions of these executables at the versions used in this project (and for which testing has been performed/reported data are valid).

The approach would look something like this:

```
$ git clone https://github.com/chrissimpkins/Crunch.git
$ cd Crunch

# build dependencies off of system PATH
$ make install-dependencies

# install `crunch` executable on system PATH
$ make install-executable

$ cd /some/path/with/png

# Crunch the things...
$ crunch *.png
```

Feedback on this approach would be helpful.

**Comment by chrissimpkins** [comment]
See PR #18, including this comment https://github.com/chrissimpkins/Crunch/pull/18#issuecomment-378649256 for details that tie the *nix support and parallel execution support (issue #17) together into a cohesive approach

**Comment by chrissimpkins** [comment]
Would greatly appreciate any feedback that you have.  Feel free to weigh in on the appropriate thread or in PR #18 where the scripting work is happening.

**Comment by chrissimpkins** [comment]
Work is underway to support this in https://github.com/chrissimpkins/Crunch/pull/19.  This will be released as part of the v2.0.0 release of Crunch.

**Comment by chrissimpkins** [comment]
Initial implementation of *nix command line executable support for parallel image processing (along with a new command line executable crunch that can be installed with make) is now available for testing in #19. Install and usage instructions are available in the OP of the pull request thread for those who would like to help test. This will be released as part of v2.0.0.

**Comment by chrissimpkins** [comment]
v2.0.0 is released with these changes
